src/app.js

```diff
@@ -0,0 +1,65 @@
+import {FilterableTable} from './components/FilterableTable.react';
+
+import * as SharedUtils from './utils/SharedUtils';
+
+document.addEventListener('DOMContentLoaded', function(){
+  var searchObject = SharedUtils.parseUrlSearchString(),
+      filterableTables = document.getElementsByClassName('react-filterable-table'),
+      filterableTableNode,
+      filterBarConfiguration,
+      tableConfiguration,
+      filterableTableId,
+      filterbar,
+      table,
+      page,
+      filters;
+
+  for (var i = 0; i < filterableTables.length; i++) {
+
+    filterableTableNode = filterableTables[i];
+    filterBarConfiguration = filterableTableNode.querySelector('dl.filterBarConfiguration');
+    tableConfiguration = filterableTableNode.querySelector('dl.tableConfiguration');
+    filterableTableId = filterableTableNode.getAttribute('id') || i;
```

Comment:
What is a filterableTableId? and how does it map to something like ```/jobseekers```. Just trying to get my head around it.